doc/source/starting-ray.rst

```diff
@@ -12,11 +12,12 @@ What is the Ray runtime?
 Ray programs are able to parallelize and distribute by leveraging an underlying *Ray runtime*.
 The Ray runtime consists of multiple services/processes started in the background for communication, data transfer, scheduling, and more. The Ray runtime can be started on a laptop, a single server, or multiple servers.
 
-There are three ways of starting the Ray runtime:
+There are four ways of starting the Ray runtime:
 
 * Implicitly via ``ray.init()`` (:ref:`start-ray-init`)
 * Explicitly via CLI (:ref:`start-ray-cli`)
 * Explicitly via the cluster launcher (:ref:`start-ray-up`)
+* (Beta) Implicitly via the `Ray Client <ray-client.html>`
```

Comment:
Can you change this from a bullet to "You can also connect to an existing Ray runtime via Ray Client (link)"